Gems/EMotionFX/Code/EMotionFX/CommandSystem/Source/AnimGraphNodeGroupCommands.h

```diff
@@ -13,22 +13,72 @@
 #include <MCore/Source/CommandGroup.h>
 #include <EMotionFX/Source/AnimGraphNodeGroup.h>
 #include <EMotionFX/Source/AnimGraph.h>
+#include <EMotionFX/CommandSystem/Source/ParameterMixins.h>
 
 namespace CommandSystem
 {
     // adjust a node group
-    MCORE_DEFINECOMMAND_START(CommandAnimGraphAdjustNodeGroup, "Adjust anim graph node group", true)
-public:
-    static AZStd::string GenerateNodeNameString(EMotionFX::AnimGraph* animGraph, const AZStd::vector<EMotionFX::AnimGraphNodeId>& nodeIDs);
-    static AZStd::vector<EMotionFX::AnimGraphNodeId> CollectNodeIdsFromGroup(EMotionFX::AnimGraphNodeGroup* nodeGroup);
+    class CommandAnimGraphAdjustNodeGroup
+        : public MCore::Command
+        , public EMotionFX::ParameterMixinAnimGraphId
+    {
+    public:
+        AZ_CLASS_ALLOCATOR_DECL
 
-    AZStd::string                               mOldName;
-    bool                                        mOldIsVisible;
-    AZ::u32                                     mOldColor;
-    AZStd::vector<EMotionFX::AnimGraphNodeId>   mOldNodeIds;
-    bool                                        mOldDirtyFlag;
-    MCORE_DEFINECOMMAND_END
+        enum class NodeAction
```

Comment:
Just a little weird too me that CommandAnimGraphAdjustNodeGroup don't have a NONE and CommandAdjustNodeGroup does have a NONE entry. Perhaps both should be a NONE as default.